### PR TITLE
Set up a new target to connect Tchap-iOS on DEV-infra #567

### PR DESCRIPTION
**Pull request by phlniji**
#567

### CODE REVIEW

File: DevTchap/Config/BuildSettings.swift [hunk 1]
```diff
@@ -0,0 +1,423 @@
+// 
+// Copyright 2020 Vector Creations Ltd
```
Comment (giomfo) [inline DevTchap/Config/BuildSettings.swift:2]: Please use "Copyright 2022 New Vector Ltd" for new added files

**Review by giomfo — APPROVED** [review]
Approved, please see my comment about copyright for all added files

**Comment by giomfo** [comment]
@phlniji I let you see about the 2 files in conflict, but I think we don't need to commit changes on them

File: DevTchap/Config/BuildSettings.swift (hunk 1 above)
Comment (phlniji) [reply DevTchap/Config/BuildSettings.swift]: Fixed in last commit.

File: DevTchap/Config/AppIdentifiers.xcconfig [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// 
+// Copyright 2021 Vector Creations Ltd
```
Comment (giomfo) [inline DevTchap/Config/AppIdentifiers.xcconfig:2]:
```suggestion
// Copyright 2022 Vector Creations Ltd
```

File: DevTchap/SupportingFiles/DevTchap-App-Common.xcconfig [hunk 3]
```diff
@@ -0,0 +1,34 @@
+// 
+// Copyright 2020 Vector Creations Ltd
```
Comment (giomfo) [inline DevTchap/SupportingFiles/DevTchap-App-Common.xcconfig:2]:
```suggestion
// Copyright 2022 Vector Creations Ltd
```

File: DevTchap/SupportingFiles/DevTchap-App-Debug.xcconfig [hunk 4]
```diff
@@ -0,0 +1,21 @@
+// 
+// Copyright 2020 Vector Creations Ltd
```
Comment (giomfo) [inline DevTchap/SupportingFiles/DevTchap-App-Debug.xcconfig:2]:
```suggestion
// Copyright 2022 Vector Creations Ltd
```

File: DevTchap/SupportingFiles/DevTchap-App-Release.xcconfig [hunk 5]
```diff
@@ -0,0 +1,24 @@
+// 
+// Copyright 2020 Vector Creations Ltd
```
Comment (giomfo) [inline DevTchap/SupportingFiles/DevTchap-App-Release.xcconfig:2]:
```suggestion
// Copyright 2022 Vector Creations Ltd
```

File: DevTchapNSE/Common.xcconfig [hunk 6]
```diff
@@ -0,0 +1,31 @@
+// 
+// Copyright 2020 Vector Creations Ltd
```
Comment (giomfo) [inline DevTchapNSE/Common.xcconfig:2]:
```suggestion
// Copyright 2022 Vector Creations Ltd
```

File: DevTchapShareExtension/Common.xcconfig [hunk 7]
```diff
@@ -0,0 +1,34 @@
+// 
+// Copyright 2020 Vector Creations Ltd
```
Comment (giomfo) [inline DevTchapShareExtension/Common.xcconfig:2]:
```suggestion
// Copyright 2022 Vector Creations Ltd
```

**Review by giomfo — APPROVED** [review]
approved but there are 6 copyrights to update

File: DevTchap/Config/AppIdentifiers.xcconfig (hunk 2 above)
Comment (phlniji) [reply DevTchap/Config/AppIdentifiers.xcconfig]: Fixed in last commit

File: DevTchap/SupportingFiles/DevTchap-App-Common.xcconfig (hunk 3 above)
Comment (phlniji) [reply DevTchap/SupportingFiles/DevTchap-App-Common.xcconfig]: Fixed in last commit

File: DevTchap/SupportingFiles/DevTchap-App-Debug.xcconfig (hunk 4 above)
Comment (phlniji) [reply DevTchap/SupportingFiles/DevTchap-App-Debug.xcconfig]: Fixed in last commit

File: DevTchap/SupportingFiles/DevTchap-App-Release.xcconfig (hunk 5 above)
Comment (phlniji) [reply DevTchap/SupportingFiles/DevTchap-App-Release.xcconfig]: Fixed in last commit

File: DevTchapNSE/Common.xcconfig (hunk 6 above)
Comment (phlniji) [reply DevTchapNSE/Common.xcconfig]: Fixed in last commit

File: DevTchapShareExtension/Common.xcconfig (hunk 7 above)
Comment (phlniji) [reply DevTchapShareExtension/Common.xcconfig]: Fixed in last commit

**Pull request merged**